.github/workflows/bazel.yml

```diff
@@ -171,9 +171,11 @@ jobs:
           image: ${{ env.BAZEL_BASE_IMAGE }}
           # TODO: Remove work-around mount of Github workspace to /magma (https://github.com/addnab/docker-run-action/issues/11)
           options: -v ${{ github.workspace }}:/workspaces/magma/
+          # --remote_download_toplevel instead of --remote_download_minimal is
+          # needed due to https://github.com/bazelbuild/bazel/issues/9386
           run: |
             cd /workspaces/magma
-            bazel run //:check_starlark_format
+            bazel run //:check_starlark_format --remote_download_toplevel
```

Comment:
Maybe add a link to https://github.com/bazelbuild/bazel/issues/9386 in a comment.

Reply:
ok, I should just open my eyes ;)